lib/components/Editor/index.js

```diff
@@ -128,6 +128,10 @@ const Tiptap = (
     autoFocus && editor?.commands.focus("end");
   }, [editor]);
 
+  useEffect(() => {
```

Comment:
Please add a check for value as:

- Value is string
- Value is not `undefined`
- value is not `null`

Reply:
@labeebklatif is this change good enough?
```js
 typeof value === "string"
      ? editor?.commands.setContent(value)
      : editor?.commands.setContent("");
```

- This will clear the editor content when the value is set as `null` or `undefined`.
- For instance, resetting a Formik component might result in the `value` being `null`.

Reply:
This should do the job